crates/client/mempool/src/l1.rs

```diff
@@ -42,11 +42,11 @@ impl GasPriceProvider {
     }
 
     pub fn set_strk_gas_price_sync_enabled(&self, enabled: bool) {
-        self.gas_price_sync_enabled.store(enabled, Ordering::Relaxed);
+        self.strk_gas_price_sync_enabled.store(enabled, Ordering::Relaxed);
```

Comment:
I know this was not changed in this pr, but why are we using `Ordering::Relaxed` here?

Reply:
Makes sense! Might be worth adding a quick comment with what you just explained here to make things clear.